src/filters/pages.js

```diff
@@ -28,11 +28,13 @@ label: isCurrent
 category: Status
 note: Check that the page does not have an end date
 params:
-  collection:
-    type: array of pages
+  page:
+    type: page object
 */
```

Comment:
It might be useful to include a note about the shapes of objects in the filter documentation. e.g. an example of what properties are in the `page object`.

Reply:
These are defined by 11ty. Maybe just saying that would be enough?

Reply:
Yea, I think so :+1: